src/Bladeburner/Actions/Casualties.ts

```diff
@@ -0,0 +1,51 @@
+import { getRandomIntInclusive } from "../../utils/helpers/getRandomIntInclusive";
+import { OperationTeam } from "../Bladeburner";
```

Comment:
import type

Reply:
Blimey I've never been called out on this so often. Do you have import -> import type automated by any chance in your IDE? It seems like this should be covered by the linter or formatter.

Thanks for being so dilligent.

Reply:
I am a robot XD

We can't turn on lint for this, because there's way too many existing violations. A vague plan is to try to incrementally clean those up so we can enforce this automatically. (It would let us enable some better options in tsconfig)